### ISSUE TITLE
When GROUP BY has null field(s), log error to ksqlDB processing log

### ISSUE THREAD

**Issue by mikebin** [confluentinc/ksql#6080]
Version: 5.5.1

If field(s) in a `GROUP BY` clause are `null`, an error is logged to the ksqlDB server log (but _not_ the [ksqlDB processing log](https://docs.ksqldb.io/en/latest/developer-guide/test-and-debug/processing-log/)), and the record is skipped:

```[2020-08-18 07:35:44,645] ERROR Group-by column with index 0 resolved to null. The source row will be excluded from the table. (io.confluent.ksql.execution.streams.GroupByParamsFactory:90)```

To make it easier for users to detect this this issue, it would be useful to log this to the ksqlDB processing log.

**Comment by ybyzek** [comment]
> To make it easier for users to detect this this issue, it would be useful to log this to the ksqlDB processing log.

Agree. In my environment I've also run into this issue and spent a good amount of time chasing down the error.
